src/index.js

```diff
@@ -1047,6 +1050,10 @@ export default class Carousel extends React.Component {
     }
   }
 
+  reverseChildren(children, rtl) {
+    return rtl ? [...children].reverse() : children;
```

Comment:
this function should only return reversed children because it's called `reverseChildren`. It's confusing that it can return `children` also.

Reply:
Also upon further review, I don't see this method being called anywhere